src/components/domain/project/projectCard/ProjectSummaryCard.tsx

```diff
@@ -36,10 +36,16 @@ export default function ProjectSummaryCard({
     variant === PROJECT_CARD_VARIANT.ADMIN || variant === PROJECT_CARD_VARIANT.LINK_PREVIEW;
   const handleClick = () => {
```

Comment:
switch문 보다 객체를 사용하는게 어떨까요? router.push를 한 번만 호출하고, 에러처리로 기본 경로도 추가해주는 게 좋을 것 같습니다.

```suggestion
  const handleClick = () => {
    if (isCardDisabled) return;
    const routes = {
      [PROJECT_CARD_VARIANT.SEARCH_RESULT]: `/project/${projectId}`,
      [PROJECT_CARD_VARIANT.MY_PROFILE]: `/project/my/${projectId}`,
      [PROJECT_CARD_VARIANT.OTHER_PROFILE]: `/project/user/${userId}/${projectId}`,
    };
    const route = routes[variant] || '/default-route';
    router.push(route);
  };
```

Reply:
존재하지 않는 타입일 때에는 기본 경로로 이동되는 것 보다, 그냥 alert을 띄워주는게 좋을 것 같습니다!

Reply:
✅ 6fb58b0794607b2a8a7885201590a7d302448012